### ISSUE TITLE
Problems when running conformance.py

### ISSUE THREAD

**Issue by didiHe** [KhronosGroup/WebGL#3481]

**Comment by didiHe** [comment]
I download the code from website,and install the packages needed.
Then I try to run the test:
python3 conformance.py --browser-name firefox --version 2.0.0
but error happened:
![image](https://user-images.githubusercontent.com/98087129/184311540-708c1b0f-fe08-457f-8148-9a6e45a18732.png)
the OS is ubuntu18.04
the python version is Python 3.8.10

**Comment by kenrussell** [comment]
@gyagp any chance you can investigate this? Unfortunately our team's not using this harness.

**Comment by gyagp** [comment]
Firefox was not supported on Linux. I just uploaded a PR to add the support, and also fixed some compatibility issue with new Selenium as it deprecated some APIs. Sorry that I don't have enough time for all the browsers, so only Firefox and Chrome are taken care of. 
BTW, I encountered a problem with Firefox from snap, as it reported profile couldn't be loaded. I followed the steps (https://www.omgubuntu.co.uk/2022/04/how-to-install-firefox-deb-apt-ubuntu-22-04) to install a .deb file instead.
